public/js/sheets.js

```diff
@@ -0,0 +1,86 @@
+async function loadSheets() {
+  //county-cities
+  Papa.parse(
+    'https://docs.google.com/spreadsheets/d/1jNk90L1FGXt3estVzFII2-eeKQZ85RYiLKCyNe14nGg/export?format=csv&gid=0',
+    {
+      download: true,
+      header: true,
+      complete: function (results) {
+        var countyCityMap = results.data;
+        setLocalItem('countyCityMap', countyCityMap);
+      },
+    }
+  );
+
+  //household-information
+  Papa.parse(
+    'https://docs.google.com/spreadsheets/d/1jNk90L1FGXt3estVzFII2-eeKQZ85RYiLKCyNe14nGg/export?format=csv&gid=679979845',
+    {
+      download: true,
+      header: true,
+      complete: function (results) {
+        var countyHouseholdMap = results.data;
+        setLocalItem('countyHouseholdMap', countyHouseholdMap);
+      },
+    }
+  );
+}
+
+async function init() {
+  if (!getLocalItem('countyHouseholdMap') || !getLocalItem('countyCityMap')) {
+    loadSheets();
```

Comment:
shouldn't we have `await` here? 
`await loadSheets();`

Reply:
we can, but this is called when the DOM is loaded, and nothing has been selected. We don't want to block loading just to wait for sheets. The initial display doesn't really require it.

Reply:
Nice. Still learning this stuff